### PR TITLE
Add new module Calendar Event State

### PR DESCRIPTION
**Pull request by archetipo**

### CODE REVIEW

File: calendar_event_state/README.rst [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+   :alt: License: AGPL-3
+
+====================
+Calendar Event State
+====================
+
+This module extends the functionality of calendar event with  the support
+of multiple custum state, priority and kanban view
+
+A basic states :
```
Comment (eLBati) [inline calendar_event_state/README.rst]: `A basic states :` --> `3 basic states:`

File: calendar_event_state/README.rst [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+*  To plan
+*  Planned
+*  Done
+
+are setting up  by default.
+
+Usage
+=====
+
+To manage custom stetes, you need to:
```
Comment (eLBati) [inline calendar_event_state/README.rst]: `stetes` -> `states`

File: calendar_event_state/README.rst [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+* Go to *Settings > Companies > Companies > your company*.
+
+
+* Go to Tab  *Calendar Event State*
+
+Edit  or create new record, like as project task states ,you may set:
```
Comment (eLBati) [inline calendar_event_state/README.rst]: `states ,you` -> `states, you`

File: calendar_event_state/models/calendar.py [hunk 4]
```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+# © 2016 Alessio Gerace - Agile Business Group
+# © 2016 Lorenzo Battistini - Agile Business Group
+# License GPL-3.0 or later (http://www.gnu.org/licenses/gpl.html).
+
+from openerp import models, fields, api
+
+
+class CalendarEventStates(models.Model):
+    _name = 'calendar.event.states'
```
Comment (eLBati) [inline calendar_event_state/models/calendar.py]: Following odoo conventions, I would use `calendar.event.state`

File: calendar_event_state/models/calendar.py [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+    sequence = fields.Integer('Sequence', default=1)
+
+    event_ids = fields.One2many(
+        'calendar.event', 'state_id',
+        'Related Events')
+
+    fold = fields.Boolean(
+        'Folded in Kanban View',
+        help='This state is folded in the kanban view when'
+        'there are no records in that stage to display.'
```
Comment (eLBati) [inline calendar_event_state/models/calendar.py]: What does this mean? Isn't it always folded by default?

Comment (archetipo) [inline calendar_event_state/models/calendar.py]: yes

**Comment by eLBati** [comment]
@archetipo please also check pylint

```
calendar_event_state/models/__init__.py:6: [W0403(relative-import), ] Relative import 'calendar', should be 'calendar_event_state.models.calendar'
calendar_event_state/models/__init__.py:7: [W0403(relative-import), ] Relative import 'company', should be 'calendar_event_state.models.company'
```

**Comment by archetipo** [comment]
@eLBati  Thanks Done

**Comment by eLBati** [comment]
@archetipo see travis

```
ParseError: "calendar.event.states" while parsing /home/travis/build/OCA/server-tools/calendar_event_state/data/calendar_state_data.xml
```

**Comment by archetipo** [comment]
Thanks @eLBati  Done

File: calendar_event_state/data/calendar_state_data.xml [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+<!-- © 2016 Alessio Gerace - Agile Business Group
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<openerp>
+    <data noupdate="1">
+
+        <record id="event_to_plan" model="calendar.event.state">
+            <field name="name">To Plan</field>
+            <field name="sequence" eval="1"/>
+            <field name="fold" eval="True"/>
```
Comment (eLBati) [inline calendar_event_state/data/calendar_state_data.xml]: @archetipo I would set 'to plan' and 'planned' states with fold = False by default. What do you think?

File: calendar_event_state/views/calendar_view.xml [hunk 7]
```diff
@@ -0,0 +1,136 @@
...
+                                        <div><b><field name="name"/></b></div>
+                                        <div>
+                                            <t t-if="record.start_datetime.raw_value and record.start_datetime.raw_value lt (new Date())" t-set="red">oe_kanban_text_red</t>
+                                            <span t-attf-class="#{red || ''}"><i><field name="start_datetime"/></i></span>
+                                        </div>
+                                        <div class="oe_kanban_bottom_lrft">
+                                                <div class="pull-left" groups="base.group_user">
+                                                    <field name="partner_ids" widget="many2manyattendee"
+                                                                context="{'force_email':True}"
+                                                                on_change="onchange_partner_ids(partner_ids)"
```
Comment (eLBati) [inline calendar_event_state/views/calendar_view.xml]: I think on_change is not needed in kanban view

**Comment by eLBati** [comment]
A couple of last remarks, then :+1: thanks!

File: calendar_event_state/README.rst [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+*  To plan
+*  Planned
+*  Done
+
+are setting up  by default.
+
+Usage
+=====
+
+To manage custom states, you need to:
```
Comment (pedrobaeza) [inline calendar_event_state/README.rst:18]: Put this in the configuration section

File: calendar_event_state/README.rst [hunk 9]
```diff
@@ -0,0 +1,78 @@
...
+
+====================
+Calendar Event State
+====================
+
+This module extends the functionality of calendar event
+with the support of multiple configurable states (like the project tasks' ones).
+It also adds a priority field and a kanban view
+
+3 basic states :
```
Comment (pedrobaeza) [inline calendar_event_state/README.rst]: Put this in the configuration section

File: calendar_event_state/README.rst [hunk 10]
```diff
@@ -0,0 +1,78 @@
...
+
+*  To plan
+*  Planned
+*  Done
+
+are setting up  by default.
```
Comment (pedrobaeza) [inline calendar_event_state/README.rst]: are set up by default

File: calendar_event_state/data/calendar_state_data.xml (hunk 6 above)
Comment (archetipo) [inline calendar_event_state/data/calendar_state_data.xml]: @eLBati  ok n.p.

File: calendar_event_state/views/calendar_view.xml (hunk 7 above)
Comment (archetipo) [inline calendar_event_state/views/calendar_view.xml]: @eLBati  thanks..

File: calendar_event_state/README.rst (hunk 8 above)
Comment (archetipo) [inline calendar_event_state/README.rst:18]: @pedrobaeza ok thanks

**Comment by archetipo** [comment]
@pedrobaeza @eLBati  Thanks Done.

**Comment by tafaRU** [comment]
This PR is superseded by https://github.com/OCA/server-tools/pull/717
Please close it.

Thank you in advance.

**Pull request closed without merging**